tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                    "emissivity": 0.84,
+                    "emissivity_back": 0.84,
+                    "conductivity": 1.0,
+                    "dirt_correction": 1.0,
+                    "solar_diffusing": false
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Gypsum Board",
+                    "roughness": "MediumSmooth",
+                    "thickness": 0.0127,
+                    "conductivity": 0.16,
+                    "density": 800.0,
+                    "specific_heat": 1090.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Acoustic Tile",
+                    "roughness": "MediumSmooth",
+                    "thickness": 0.02,
+                    "conductivity": 0.06,
+                    "density": 368.0,
+                    "specific_heat": 590.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.3,
+                    "visible_absorptance": 0.3
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Metal Surface",
+                    "roughness": "Smooth",
+                    "thickness": 0.0015,
+                    "conductivity": 45.0,
+                    "density": 7690.0,
+                    "specific_heat": 410.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Thick Stone",
+                    "roughness": "Rough",
+                    "thickness": 0.3,
+                    "conductivity": 2.31,
+                    "density": 2322.0,
+                    "specific_heat": 832.0,
+                    "thermal_absorptance": 0.95,
+                    "solar_absorptance": 0.75,
+                    "visible_absorptance": 0.8
+                }
+            ]
+        }
+    },
+    "rooms": [
+        {
+            "type": "RoomAbridged",
+            "name": "TinyHouseZone",
+            "name_original": "Tiny House Zone",
+            "properties": {
+                "type": "RoomProperties",
+                "energy": {
+                    "type": "RoomEnergyPropertiesAbridged",
+                    "program_type": null,
+                    "construction_set": null,
+                    "people": null,
+                    "lighting": null,
+                    "electric_equipment": null,
+                    "gas_equipment": null,
+                    "infiltration": null,
+                    "ventilation": null
+                }
+            },
+            "faces": [
```

Comment:
@chriswmackey is there a need for a validation in the schema to ensure the number of faces are 6?

Reply:
I assume there can be more than 6 faces.  Minimum number of faces for an enclosed zone would be 4, might be best to have no limits to allow unenclosed zones.

Reply:
> might be best to have no limits to allow unenclosed zones

@macumber curios to know what are the cases that you think an unenclosed zone will be useful? We are currently enforcing closed zones in the UI.

Reply:
Some applications (e.g. CBECC-com) use pinwheel models for some applications.  You can't use all the shadowing methods and you have to manually input zone volumes.  I think it's just an application preference, if your users don't need it then I think requiring enclosed zones is good.

Reply:
@macumber ,
The Python objects for Room geometry have checks for solidity and, because of this, we intend to enforce closed volumes for all of the models created from the plugins. However, I would agree that we don't need to be strict in enforcing closed volumes within the schema validation.  People may have a model without closed volumes that they intend to edit into one that has closed volumes (I'm thinking about the case where Theo makes an editor for models in the honeybee JSON schema).  Also, if a user of Pollination API wants us to run a check for solidity, we can set up a service that re-serializes the JSON to honeybee Python objects and runs the solidity check.

Reply:
I am going to resolve this now that we agreed that keeping the schema felxible is a good strategy.